charts/webapp/Chart.yaml

```diff
@@ -5,4 +5,4 @@ maintainers:
   - name: DAI
     email: dai@tamedia.ch
```

Comment:
I don't think we want to expose our team email? (well it is now since its in this comment)

Also this is just a placeholder to keep the linter happy